packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -585,6 +585,9 @@ export class IndexedDbPersistence implements Persistence {
   private canActAsPrimary(
     txn: PersistenceTransaction
   ): PersistencePromise<boolean> {
+    if (this.forceOwningTab) {
+      return PersistencePromise.resolve<boolean>(true);
```

Comment:
You can probably drop the generic since it can be inferred.

Reply:
Not sure why TS can't infer it, but the error I get is:
`Type 'PersistencePromise<true>' is not assignable to type 'PersistencePromise<boolean>'.`